web/yo/app/scripts/services/mainutils.js

```diff
@@ -644,6 +644,34 @@ angular.module('oncokbApp')
                 console.log(error);
             });
         }
+        function getString(string) {
+            if(!string || !_.isString(string)) {
+                return '';
+            }
+            var tmp = window.document.createElement('DIV');
+            var processdStr = string.replace(/(\r\n|\n|\r)/gm, '');
+            var processdStr = processdStr.replace(/<style>.*<\/style>/i, '');
+            tmp.innerHTML = processdStr;
+            /* eslint new-cap: 0*/
+            var _string = tmp.textContent || tmp.innerText || S(string).stripTags().s;
+            string = S(_string).collapseWhitespace().s;
+            string = string.replace(/<!--.*-->/g, '');
+
+            // Convert the html fragment again to trim the reserved text.
+            var tmp = window.document.createElement('DIV');
+            tmp.innerHTML = string;
+
+            var _string = tmp.textContent || tmp.innerText || S(string).stripTags().s;
+            string = S(_string).decodeHTMLEntities().s;
```

Comment:
decodeHTMLEntities() can help to convert html symbol to characters(like `&nbsp;` -> ` `) but remove html tags, so I inserted decodeHTMLEntities() in getString() and didn't change other code.